rest/taskana-rest-spring-example-boot/pom.xml

```diff
@@ -55,6 +55,21 @@
       <artifactId>springdoc-openapi-starter-webmvc-ui</artifactId>
       <version>${version.spring-openapi}</version>
     </dependency>
+    <dependency>
```

Comment:
Aren't those dependencies for local testing only? If they are, please remove them.

Reply:
Yes you are right, I will remove them